Documentation/helm-values.rst

```diff
@@ -3188,6 +3188,10 @@
      - Configure what the response should be to traffic for a service without backends. "reject" only works on kernels >= 5.10, on lower kernels we fallback to "drop". Possible values:  - reject (default)  - drop
      - string
      - ``"reject"``
+   * - :spelling:ignore:`skipKernelModuleValidation`
+     - 
```

Comment:
Why isn't `make -C Documentation helm-values.rst` copying over the comment from `install/kubernetes/cilium/values.yaml.tmpl`? 🤔

Reply:
Did you try `make -C Documentation update-helm-values` instead?

Reply:
Does not make a difference:
```bash
$ make -C Documentation update-helm-values
docker container run --rm --workdir /src/Documentation --env GIT_CONFIG_COUNT=1 --env GIT_CONFIG_KEY_0=safe.directory --env GIT_CONFIG_VALUE_0=/src --volume /Users/phisch/git/github.com/philipschmid/cilium/Documentation/..:/src --user "501:20" --env READTHEDOCS_VERSION=latest --env SKIP_LINT= --env INCREMENTAL= "quay.io/cilium/helm-toolbox:"v1.1.0"@sha256:"961693f182b9b456ed90e5274ac5df81e4af4343104e252666959cdf9570ce9e"" helm-docs -d -c /src/install/kubernetes -t /src/Documentation/helm-values.tmp.tmpl > helm-values.tmp
time="2024-11-06T14:16:17Z" level=info msg="Found Chart directories [cilium]"
time="2024-11-06T14:16:17Z" level=info msg="Generating README Documentation for chart /src/install/kubernetes/cilium"
awk -F'|' '{print "|"$2"|"$5"|"$3"|"$4"|"}' helm-values.tmp > helm-values.awk
docker container run --rm --workdir /src/Documentation --env GIT_CONFIG_COUNT=1 --env GIT_CONFIG_KEY_0=safe.directory --env GIT_CONFIG_VALUE_0=/src --volume /Users/phisch/git/github.com/philipschmid/cilium/Documentation/..:/src --user "501:20" --env READTHEDOCS_VERSION=latest --env SKIP_LINT= --env INCREMENTAL= "quay.io/cilium/helm-toolbox:"v1.1.0"@sha256:"961693f182b9b456ed90e5274ac5df81e4af4343104e252666959cdf9570ce9e"" python3 /usr/bin/m2r2 --overwrite helm-values.awk
gsed 's/^\(   \* - \)\([[:print:]]\{1,\}\)$/\1:spelling:ignore:`\2`/' helm-values.rst > helm-values.sed
printf '..\n  %s\n\n%s\n' "AUTO-GENERATED. Please DO NOT edit manually." "$(cat helm-values.sed)" > helm-values.rst
rm -f -- helm-values.tmp helm-values.awk helm-values.sed
$ git status
On branch pr/philip/skip_kernel_module_validation_loading
Your branch is up to date with 'origin/pr/philip/skip_kernel_module_validation_loading'.

nothing to commit, working tree clean
```

I tried it on macOS and Linux (Ubuntu 24.04).

Reply:
Found it, this is because you have a non-breakable space at the beginning of your line here:

```
# -- Skip Cilium's kernel module validation. Only enable this flag if all
 ^ here
```

Just replace with a normal space (if you don't drop the PR).

Reply:
Oh wow, nice catch! Thanks 🙏.

Reply:
BTW, in case somebody else also does not want to run into such issues anymore: https://stackoverflow.com/a/58854106
https://marketplace.visualstudio.com/items?itemName=viktorzetterstrom.non-breaking-space-highlighter&ssr=false#overview
![image](https://github.com/user-attachments/assets/16d186ef-0c79-4a2e-97c4-e8dc61c712bc)